rand_core/src/block.rs

```diff
@@ -188,6 +188,7 @@ where <R as BlockRngCore>::Results: AsRef<[u32]> + AsMut<[u32]>
         let read_u64 = |results: &[u32], index| {
             if cfg!(any(target_endian = "little")) {
                 // requires little-endian CPU
+                #[allow(clippy::cast_ptr_alignment)]  // false positive
                 let ptr: *const u64 = results[index..=index+1].as_ptr() as *const u64;
```

Comment:
`results` is aligned to 32bits, but then 64bits are read. I don't understand why this would be a false positive.

Reply:
https://github.com/rust-lang/rust-clippy/issues/2881

Guess what we do with the result: `ptr::read_unaligned(ptr)`